src/web/src/json.ecpp

```diff
@@ -60,12 +60,13 @@
 	static cxxtools::Regex rex_licensing ("^/api/v1/admin/licensing$");
 	static cxxtools::Regex rex_getlog ("^/api/v1/admin/getlog/.*$");
 	static cxxtools::Regex rex_sse ("^/api/v1/sse.*$");
+        static cxxtools::Regex rex_getaverage ("^/api/v1/metric/computed/average/.*$");
 
 	if(request.getMethod() == "GET") {
 		log_debug("%s", ("GetURL==" + request.getUrl()).c_str() );
         std::string url = request.getUrl ();
 
-		if (rex_license.match(url) || rex_export.match (url) || rex_licensing.match (url) || rex_getlog.match (url) || rex_sse.match (url))
+		if (rex_license.match(url) || rex_export.match (url) || rex_licensing.match (url) || rex_getlog.match (url) || rex_sse.match (url) || rex_getaverage.match (url))
```

Comment:
If we don't set content type to json here, who will? average.ecpp sets content type only for CSV, so just checking who sets the proper type in the other cases? Thanks